appserver/distributions/glassfish-common/src/main/resources/bin/asadmin

```diff
@@ -15,17 +16,15 @@
 # SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
 #
 
-AS_INSTALL=`dirname "$0"`/../glassfish
+export AS_INSTALL="$(dirname "$(realpath -s "$0")")/../glassfish"
 case "`uname`" in
   CYGWIN*) AS_INSTALL=`cygpath --windows $AS_INSTALL`
 esac
-AS_INSTALL_LIB="$AS_INSTALL/lib"
 . "${AS_INSTALL}/config/asenv.conf"
+
 JAVA=java
-#Depends upon Java from ../config/asenv.conf
 if [ ${AS_JAVA} ]; then
```

Comment:
Can we keep the comment "Depends upon Java from ../config/asenv.conf" ?

Reply:
The comment was false.